crates/revm/src/db/ethersdb.rs

```diff
@@ -1,10 +1,44 @@
-use crate::primitives::{AccountInfo, Address, Bytecode, B256, KECCAK_EMPTY, U256};
-use crate::{Database, DatabaseRef};
+use std::sync::Arc;
+
 use ethers_core::types::{BlockId, H160 as eH160, H256, U64 as eU64};
 use ethers_providers::Middleware;
-use std::sync::Arc;
 use tokio::runtime::{Builder, Handle, RuntimeFlavor};
 
+use crate::primitives::{AccountInfo, Address, Bytecode, B256, KECCAK_EMPTY, U256};
+use crate::{Database, DatabaseRef};
+
+/// internal utility function to call tokio feature and wait for output
+#[inline]
+fn block_on<F>(f: F) -> F::Output
```

Comment:
As this is only used by `EthersDB` there is no reason to be outside of it, can you revert back this change